### PR TITLE
Admin user controll

### PR DESCRIPTION
**Pull request by MarcvanMelle**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,7 +1,50 @@
...
+    else
+      render(file: File.join(Rails.root, 'public/403.html'), status: 403, layout: false)
+    end
+  end
+
   def show
     @user = User.find(params[:id])
   end
+
+  def destroy
```
[the remark this answers is not in the document]
Comment (MarcvanMelle) [reply app/controllers/users_controller.rb:16]: Fixed

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,7 +1,54 @@
 class UsersController < ApplicationController
   before_action :authenticate_user!
 
+  def index
+    if current_user.try(:admin?)
+      @users = User.all.order(created_at: :asc)
+    else
+      render(file: File.join(Rails.root, 'public/403.html'), status: 403, layout: false)
```
Comment (alxjrvs) [inline app/controllers/users_controller.rb:8]: iiiiIIIIIIiiiiinteresting. This is an unconventional way to do this, but is totally valid. Consider breaking some of this into private methods to make it less of a giant line.

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -1,7 +1,54 @@
...
+
   def show
     @user = User.find(params[:id])
   end
+
+  def destroy
+    @user = User.find(params[:id])
+    if authorized_party
+      if @user.valid?
+        user_valid_destroy
```
Comment (alxjrvs) [inline app/controllers/users_controller.rb:20]: Nice encapsulation.

File: spec/features/19_admin_deletes_user_spec.rb [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  let!(:safe_word) { FactoryGirl.create(:word, user: users[3]) }
+  let!(:safe_review) { FactoryGirl.create(:review, user: users[2], word: safe_word) }
+  let!(:reviews) { FactoryGirl.create_list(:review, 4, user: users[0], word: words[0]) }
+  let!(:reviews) { FactoryGirl.create_list(:review, 4, user: users[0], word: safe_word) }
+
+  context "As an admin" do
+    scenario "I can delete a user from the user's profile page" do
+      login_as(admin, scope: :user)
+      visit user_path(users[0])
+      deleted_username = users[0].username
```
Comment (alxjrvs) [inline spec/features/19_admin_deletes_user_spec.rb:16]: You can do this in a `let` too - `lets` can hold any arbitrary code. 

`let!(:deleted_username) { users.first.username}`

**Review by alxjrvs — APPROVED** [review]
This LGTM. Great work!

**Pull request merged**